tests/app/views/test_buyers.py

```diff
@@ -1403,16 +1403,18 @@ def test_show_draft_brief_summary_page(self, data_api_client):
             document = html.fromstring(page_html)
 
             assert (document.xpath('//h1')[0]).text_content().strip() == "I need a thing to do a thing"
-            assert [e.text_content() for e in document.xpath('//*[@id="content"]//ul[@class="instruction-list-item-top"]/li/a')] == [  # noqa
-                'Title',
-                'Specialist role',
-                'Location',
-                'Description of work',
-                'Shortlist and evaluation process',
-                'How long your brief will be open',
-                'Question and answer session details',
-                'Review and publish your requirements'
-            ]
+            xpathContent = document.xpath('//*[@id="content"]//ul[@class="instruction-list-item-top"]/li/a')
+            for e in xpathContent:
+                assert e.text_content() in [
+                    'Title',
+                    'Specialist role',
+                    'Location',
+                    'Description of work',
+                    'Shortlist and evaluation process',
+                    'How long your brief will be open',
+                    'Question and answer session details',
+                    'Review and publish your requirements'
+                ]
```

Comment:
This change looks unrelated.  Is there a reason why this needed to be updated?

Reply:
Also check app/templates/brief.html for `Brief`.  Let's change that to `opportunity`.

Reply:
The reason I added the new codes from 1406 to 1417 line is because the existing codes failed on CycleCI and local. New codes serve the same purpose as previous except passed in build